sse2neon.h

```diff
@@ -2442,7 +2499,11 @@ FORCE_INLINE __m128 _mm_setzero_ps(void)
 // https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_sfence
 FORCE_INLINE void _mm_sfence(void)
 {
+#if defined(_MSC_VER)
+    __dmb(_ARM64_BARRIER_SY);
```

Comment:
I don't know if this actually the best choice. This just happens to be how the MemoryBarrier() macro in winnt.h is defined.

To reproduce what _sse2neon_smp_mb() does, 
 the barrier type would be _ARM64_BARRIER_ISH. Again, I don't know if this is actually the best choice.

Reply:
I'll swap to _ARM64_BARRIER_ISH to maintain consistency - if it needs to be changed for both implementations, that can be a different discussion elsewhere